examples/gnrc_tcp_cli/Makefile

```diff
@@ -10,6 +10,21 @@ PORT = tap0
 #DEFAULT_CHANNEL ?= 17
 #CFLAGS += -DDEFAULT_CHANNEL=$(DEFAULT_CHANNEL)
 
+# Mark Boards with insufficient memory
+BOARD_INSUFFICIENT_MEMORY := airfy-beacon arduino-duemilanove \
+                             arduino-mega2560 arduino-uno \
+                             calliope-mini chronos \
+														 microbit sb-430 \
+														 sb-430h nrf51dongle \
+														 nrf6310 nucleo-f030 \
+														 nucleo-f042 nucleo-f070 \
+														 nucleo-f072 nucleo-f334 \
+														 pca10000 pca10005 \
+														 stm32f0discovery telosb \
+														 weio wsn430-v1_3b \
+														 wsn430-v1_4 yunjia-nrf51822 \
+														 z1
```

Comment:
Please use spaces instead of tabs. And you have can make your lines a little wider (up to 80 chars) ;-)